test/format/salvage.c

```diff
@@ -87,20 +88,20 @@ corrupt(void)
 
     offset = mmrand(NULL, 0, (u_int)sb.st_size);
     len = (size_t)(20 + (sb.st_size / 100) * 2);
-    testutil_check(__wt_snprintf(buf, sizeof(buf), "%s/SALVAGE.corrupt", g.home));
-    if ((fp = fopen(buf, "w")) == NULL)
-        testutil_die(errno, "salvage-corrupt: open: %s", buf);
+    testutil_check(__wt_snprintf(path, sizeof(path), "%s/SALVAGE.corrupt", g.home));
+    if ((fp = fopen(path, "w")) == NULL)
+        testutil_die(errno, "salvage-corrupt: open: %s", path);
     (void)fprintf(fp, "salvage-corrupt: offset %" PRIuMAX ", length %" WT_SIZET_FMT "\n",
       (uintmax_t)offset, len);
     fclose_and_clear(&fp);
 
     if (lseek(fd, offset, SEEK_SET) == -1)
         testutil_die(errno, "salvage-corrupt: lseek");
 
-    memset(buf, 'z', sizeof(buf));
+    smash = "!!! memory corrupted by format to test salvage ";
```

Comment:
Why is it called `smash`?

Reply:
I'm gonna say because it smashes the previously stored value, and sounds cool.